moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -448,6 +445,42 @@ void MotionPlanningFrame::importResource(const std::string& path)
         return;
       }
 
+      // If the object is very large, ask the user if the scale should be reduced.
+      bool object_is_very_large = false;
+      for (unsigned int i = 0; i < mesh->vertex_count; i++)
+      {
+        if ((abs(mesh->vertices[i * 3 + 0]) > 10) || (abs(mesh->vertices[i * 3 + 1]) > 10) ||
```

Comment:
Just, checking for components larger than 10 is not sufficient. What if a correctly scaled object is just shifted by 10 units? As scaling can be done easily in the rviz gui as well, I actually don't see the need for this extra code here.

Reply:
I doubt that many correctly scaled objects are defined 10 meters away from their origin. It sounds like a rare case - surely these users can live with clicking "No" in a dialogue window. 

The more common case is a user loading an STL that is defined in millimeters rather than meters, and then having to scale it by exactly `0.001`. Forcing the user to do that with a slider isn't only bad UX, but it doesn't even work: it leaves a gigantic interactive marker in the scene. Feel free to try with [this file](https://github.com/ros-planning/geometric_shapes/files/4741236/03-PANEL2.zip).

Anyway, in my opinion it's 100% worth checking for this, and I am fine with the check not being perfect, since it only triggers a user dialogue.

Reply:
can you make 10 a named constant such as LARGE_MESH_THRESHOLD?

Reply:
Done

Reply:
I'll resolve this for now